src/boot/efi/stub.c

```diff
@@ -151,6 +151,10 @@ static bool use_load_options(
         if (secure_boot_enabled() && have_cmdline)
                 return false;
 
+        /* Disable custom command lines if we are in confidential VMs */
+        if (secure_boot_enabled() && is_confidential_vm())
```

Comment:
Let's not call `secure_boot_enabled()` twice. (Or make it caching.)

Reply:
sounds sensible, just move the check in the if above

Reply:
hmm, shouldn't the secure_boot_enabled() part of the check be dropped? i.e. should we ever accept this in a cvm?

Reply:
I think it should be left: breaking glass to debug is always good to have. The important thing is that security is 'off' when that happens. A cvm is pretty much a hardware + firmware configuration, not sure it can be really 'turned off'.

Reply:
so lemme ask this differently: is cvm without sb a thing? do we need to bother?

@berrange what's your take on this btw?

Reply:
I think so, again for debugging/development purposes, not production

Reply:
I wouldn't expect a CVM to run in production without SB enabled, but it is technically possible to have this combination - Azure for example allows 'SecureBoot' as a user toggle option during provisioning of their current CVM support with SNP.